src/notebook/path helpers.jl

```diff
@@ -28,7 +28,8 @@ function maybe_convert_path_to_wsl(path)
     try
         if detectwsl()
             # wslpath utility prints path to stderr if it fails to convert
-            # (it fails for WSL-valid paths)
+            # (it used to fail for WSL-valid paths)
+            startswith(path, "/mnt/c/") && return path
```

Comment:
```suggestion
             isnothing(match(r"^/mnt/\w+/", path)) && return path
```

Reply:
Just tested the new change and this did work on my system.  Thanks!